Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'http://rubygems.org'
+
+gem 'sinatra', '~>1.4.7'
+gem 'sqlite3', '~>1.3.11'
+gem 'rerun', '~>0.11.0'
+gem 'pry', '~>0.10.3'
```

Comment:
I'm not sure if you all have learned about this yet, but consider putting your Gems into groups (Development, Production, Testing) to ensure you only load the gems you need for the environment you are in. Here is a resource that might help explain: http://bundler.io/v1.3/groups.html